userspace/libsinsp/events/sinsp_events.h

```diff
@@ -206,21 +221,38 @@ set<ppm_event_code> sinsp_state_event_set();
 
 /*!
   \brief Get the name of all the events provided in the set.
+  Note:
+
+  When setting resolve_sc to false exact event table names are returned for each
+  ppm_event w/ exception of "syscall" and "unknown" placeholder strings.
+
+  When setting resolve_sc to true each ppm_event will be resolved to its entirety of true
+  syscall string names, which can result in more syscalls in the following cases where
+  there is no 1:1 mapping from event code to syscall:
+
+  e.g. overloaded case: accept -> accept, accept4
+  e.g. ppm_event sharing: eventfd or eventfd2 -> always both eventfd, eventfd2
+  e.g. snowflake cases: umount or umount2 -> always both umount, umount2
+  e.g. generic events -> will map to ALL generic syscalls (over 200 generic syscalls)
 */
-std::unordered_set<std::string> event_set_to_names(const set<ppm_event_code>& events_set);
+std::unordered_set<std::string> event_set_to_names(const set<ppm_event_code>& events_set, bool resolve_sc = true);
```

Comment:
```suggestion
std::unordered_set<std::string> event_set_to_names(const set<ppm_event_code>& events_set, bool resolve_generic = true);
```
Why resolving the converted SC set, instead of just resolving generic events? I feel like we're creating an equivalento of `sc_set_to_names`. What I would expect from this function is to treat generics as `syscall` or as the set of actual generic event names, depending on the boolean. WDYT?

Reply:
Incorporated :)